### PR TITLE
feat: add diagnostics instead of deadline-miss topic

### PR DESCRIPTION
**Pull request by xygyo77**

### CODE REVIEW

File: src/tilde_timing_monitor/src/tilde_timing_monitor_core.cpp [hunk 1]
```diff
@@ -511,28 +520,47 @@ bool TildeTimingMonitor::isValidDeadlineTimer(TildePathConfig & pinfo, DeadlineT
...
+  // RCLCPP_INFO(get_logger(), "[%.6f]--[%s]:%04d called", get_now(), __func__, __LINE__);
+  using diagnostic_msgs::msg::DiagnosticStatus;
+
+  auto diag_period = updater_->getPeriod().seconds();
+  bool warn_f = false;
+  bool error_f = false;
+  dbg_info_->cbStatisEnter(__func__);
+
+  for (auto & pinfo : required_paths_map_.at(params_.mode)) {
+    std::lock_guard<std::mutex> lock(*pinfo.tm_mutex);
```
[the remark this answers is not in the document]
Comment (xygyo77) [reply src/tilde_timing_monitor/src/tilde_timing_monitor_core.cpp]: I understand

**Comment by xygyo77** [comment]
> @xygyo77 I'd like to change these debagu command to easy words. Please let me discuss about wordings separately.
> 
> Unknown word (clrinfo) Unknown word (deton) Unknown word (detoff) Unknown word (dbgon) Unknown word (dbgoff) Unknown word (histon) Unknown word (histoff) Unknown word (dispon) Unknown word (dispoff)

I think so too.

File: src/tilde_timing_monitor/src/tilde_timing_monitor_core.cpp [hunk 2]
```diff
@@ -529,28 +537,47 @@ bool TildeTimingMonitor::isValidDeadlineTimer(TildePathConfig & pinfo, DeadlineT
...
+  // RCLCPP_INFO(get_logger(), "[%.6f]--[%s]:%04d called", get_now(), __func__, __LINE__);
+  using diagnostic_msgs::msg::DiagnosticStatus;
+
+  auto diag_period = updater_->getPeriod().seconds();
+  bool warn_f = false;
+  bool error_f = false;
+  dbg_info_->cbStatisEnter(__func__);
+
+  for (auto & pinfo : required_paths_map_.at(params_.mode)) {
+    //std::lock_guard<std::mutex> lock(*pinfo.tm_mutex);
```
Comment (nabetetsu) [inline src/tilde_timing_monitor/src/tilde_timing_monitor_core.cpp:552]:
```suggestion
    
```

Comment (xygyo77) [reply src/tilde_timing_monitor/src/tilde_timing_monitor_core.cpp]: I got it.

**Review by nabetetsu — APPROVED** [review]
LGTM :+1:

**Pull request merged**